### ISSUE TITLE
[Feature]: Settings backup support

### ISSUE THREAD

**Issue by NikunjKhangwal** [fast4x/RiMusic#1914]
### Description

Including settings preference in backup.

It's a expansion of this discussion #1911

### Suggested Solution

*

**Comment by ikanakova** [comment]
And it would be cool to have an option to automatically save to a folder on the device. (Because I think it probably wouldn't be as easy to implement in the cloud, and might require some non-free or tracking services).

**Comment by NikunjKhangwal** [comment]
Agreed. I also save my apps data in a single folder which automatically gets synced to the cloud daily. Auto backup schedule would be nice someday in future.
